src/kernels/composable_kernel/include/utility/config.hpp

```diff
@@ -11,6 +11,23 @@
 // device backend
 #define CK_DEVICE_BACKEND_AMD 1
 
+// hardware instruction support list
+#ifndef CK_HARDWARE_SUPPORT_AMD_V_MAC_F32
+#define CK_HARDWARE_SUPPORT_AMD_V_MAC_F32 0
```

Comment:
@asroy CK_HARDWARE_SUPPORT_AMD_V_MAC_F32 = 0? Why?

Reply:
v_mac is not supported on all hardware. So we should turn it off by default.

I should have but forgot to turn it on again in solver